src/api/payments/index.ts

```diff
@@ -26,6 +26,13 @@ export const getTemporaryTakeout = async (body: TakeoutTemporaryRequest) => {
   return response;
 };
 
+export const getPaymentInfo = async (paymentId: number) => {
+  const response = await apiClient.get<ConfirmPaymentsResponse>(`payments/${paymentId}`, {
+    headers: getAuthHeader(),
+  });
+  return response;
+};
```

Comment:
통일성을 위해서 params에 paymentId를 넣어주면 좋을 거 같아요!
```suggestion
export const getPaymentInfo = async (paymentId: number) => {
  const response = await apiClient.get<ConfirmPaymentsResponse>(`payments`, {
    headers: getAuthHeader(),
    params: {
      paymentId
    }
  });
  return response;
};
```

Reply:
감사합니다! 수정했습니다

[c9a7362](https://github.com/BCSDLab/KOIN_ORDER_WEBVIEW/pull/132/commits/c9a736271fcbda796eb008aee80f09bdeb47ce96)